### DISCUSSION TITLE
Midi loop back

### DISCUSSION THREAD

**Discussion by ptowns** [bangcorrupt/hacktribe#275] [General]
Lately I have been exploring Rolands MC101 and its "zen-core" sound engine.  It uses what they call partials, essentially 4 voices summed together that can be controlled independently or synchronized. I have implemented the same technique on the E2 by simply creating a sound and sequence, then copying part 3 times. Then I slightly tweak the 4 parts and hit play to create a 4 stacked voice. 
Curious if there would be a way to hack the midi implementation to back feed midi,  channel 1 to 2,3,4. There for if you play a note on 1 it would play all 4.  I'm guessing not possible, but a round robbin mode would be even better, where a single note alternates tracks. I think this would greatly expand the synth capabilities of the E2. I could easily build a puredata patch to do this, but would be cool to have it build in or 'hacked' into the device.

**Comment by bangcorrupt** [comment]
Yes, I want this too, but I haven't started any work on it.  It would be great for drum synthesis like the Nord Drum, or layering up neuro bass.  

I think it could be implemented by linking the sequencers and controls across parts, with a round robin note on option for polyphony.  We could define parameter links with offsets and inversions, like using opposing pan controls for wideness, or offset filter cutoffs for a crossover.    

I'd also like to add a macro control layer, so we can map multiple parts and parameters to single controls, and jam the whole pattern from a single surface.  We might have to rely on external MIDI control for all this for a while.

**Reply by 777paranoia** [reply]
Second this. This would change the game. I've been doing the same thing with copying the parts, been dreaming of being able to do this for years.
